From c25babba417e30166fa0dd052f8f379b7e016ac2 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 11 Apr 2024 16:08:17 -0700
Subject: [PATCH] ==== test DAMOS quota goal ====

Subject: [PATCH] selftests/damon: add DAMOS quota goal test

Implement and add a simple selftest for DAMOS quota goal feature.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/fxg59jjv | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/fxg59jjv

diff --git a/damon_meta_changes/fxg59jjv b/damon_meta_changes/fxg59jjv
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.2

